<a href="https://colab.research.google.com/github/TUIlmenauAMS/ClimateChangeCalculated/blob/main/klimawandelNachgerechnet5.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# <center>  Klimawandel Nachgerechnet 5
## <center> Wie viel Platz brauchen wir für erneuerbare Energie?
### <center> Prof. Dr.-Ing. G. Schuller
### <center> Technical University Ilmenau
### <center> Institute for Media Technology


### Dieses Mal beantworten wir die Frage: Wie viel Platz brauchen wir?

In Teil 4 haben wir die Frage beantwortet: Was können wir tun?
* Dort sahen wir, dass erneuerbare Energie effizient und billig genug ist, um mit fossiler Energie zu konkurrieren und sie zu ersetzen.
* Jetzt beantworten wir die Frage: Wie viel Platz wird sie benötigen?


## Klima:
* Wir hatten 2019/20 einen ungewöhnlich warmen Winter, fast ohne Schnee und Frost.
* Während der Corona-Restriktionen wurde die Luft merklich klarer. 
* Weniger Verschmutzung, mehr Sonnenschein.
* Dies zeigt die kurzfristigen Auswirkungen der Luftverschmutzung (und deren Beseitigung)
* Im Winter 20/21 hatten wir aufeinander folgende Extreme: Extreme Kälte und Schneefall, dann ungewöhnliche Wärme im Februar






* Ende Frühjahr und Sommer 2021 gab es schon extreme Temperaturen im Nordwesten der USA und in West-Kanada. Dort wurden fast 50 C erreicht, in Gegenden die normalerweise zu der Zeit unter 30C haben!
* In Deutschland wechseln sich hohe Temperaturen und dauerhafter extremer Starkregen mit Überflutungs-Katastrophen wie im Westen Deutschlands ab.
* Rekordwerte werden global nicht nur etwas übertroffen, sondern gleich erheblich! Ein Kennzeichen des Klimawandels.


* Pandemien wie Corona Pandemie werden häufiger, als Folge des Klimawandels, weil durch ihn Habitate verschoben werden und neue Viren öfter auf neue Spezien überspringen.
* Die MunicRe Rückversicherung berechnet die stark zunehmenden Kosten durch Wetterbedingte Naturkatastrophen von 1980 bis 2019:
https://www.munichre.com/de/risiken/extremwetter.html
* Das deutet nochmal auf die in meinen vorherigen Videos gezeigte Dringlichkeit, den globalen fossilen CO2 Ausstoss auf Null zu reduzieren. 

## Was ist die Benötigte Fläche für erneuerbare Energien in Deutschland?
* Gesamter Stromverbrauch von 2002 bis 2019 (dem Vor-Corona Jahr) und Anteil der Energiequellen:
* [Fraunhofer ISE, Stromerzeugung 2019](https://www.ise.fraunhofer.de/content/dam/ise/de/documents/news/2019/Stromerzeugung_2019_2.pdf)
* Das Diagramm auf S. 14 zeigt: Von 2002 bis 2019 nur ein geringer Anstieg von Stromerzeugung und -verbrauch.
* Der Stromverbrauch hat in den letzten Jahren teils sogar **abgenommen**!

* Für eine Schätzung auch der Zukunft könnten wir daher die Werte von 2019 verwenden.
* 2019: Stromerzeugung insgesamt (S. 13) **515 TWh**.
* Davon erneuerbare Energiequellen : 237 TWh, 
* davon 127 TWh Wind- und 47 TWh Solarenergie.




* Es gab jedoch kürzlich (Juli 2021) eine Abschätzung des Stromverbrauchs in 2030 durch die Bundesregierung:
[BMWI](https://www.bmwi.de/Redaktion/DE/Pressemitteilungen/2021/07/20210713-erste-abschaetzungen-stromverbrauch-2030.html)
* Ergebnis: "der Mittelwert der Prognose liegt bei **655 Terawattstunden**. Unterstellt werden dabei u.a. 14 Mio. Elektro-Pkw, 6 Mio. Wärmepumpen und 30 Terawattstunden Strom für grünen Wasserstoff." 
* D.h. wir brauchen bis 2030 die Differenz des progostizierten Verbrauchs zur gegenwärtig erzeugten erneuerbaren Energie, 655 - 237 = **418 TWh** zusätzlich.

**Notation:** 
* **1 kWh** (Kilo Watt Stunde) = 1000 Wh = 10³ Wh =1e3 Wh
* **1 TWh** (Terra Watt Stunde) = 10¹² Wh = 1e12 Wh


## Ertragsrisiken von erneuerbaren Energien
* Erneuerbare Energien sind am zuverlässigsten bei Verteilung auf **verschiedene Energiequellen, wie Wind und Sonne**,
* auch als saisonaler Ausgleich
* und bei geographischer Diversität, also der Nutzung des **Europäischen Stromverbundnetzes**.
* Siehe auch: [Deutscher Wetter Dienst, Pressemitteilung 2018-03-06 Klima](https://www.dwd.de/DE/presse/pressemitteilungen/DE/2018/20180306_pressemitteilung_klima_pk_news.html)
* Daher meine **Annahme:** *Photovoltaik und Windenergie sollen jeweils 50% übernehmen*, also je 418/2= **209 TWh/Jahr**, 


## Energieertrag pro Quadratmeter Solarzellen:

In [ ]:
%%html
<iframe src="https://www.energie-experten.org/erneuerbare-energien/photovoltaik/planung/ertrag.html#c27405" width="900" height="900"></iframe>

* Also Globalstrahlung in Deutschland: Ca. **1000 kWh/qm pro Jahr**
* Wirkungsgrad von modernen Solarzellen: ca. 20%
* Also Energie pro Fläche pro Jahr: Ca. **200 kWh/qm pro Jahr**.

<!-- * Die nötige Fläche berechnet sich zu: benötigte Energie pro Jahr / Energie pro Fläche pro Jahr. -->
<!-- * Wir nutzen "Python Sympy" für das Rechnen mit Einheiten, auch als Prüfung der Richtigkeit der Rechnung: -->

## Flächenbedarf Solarzellen für 209 TWh pro Jahr
Flaechenbedarf = Energie pro Jahr geteilt durch Energie pro Fläche pro Jahr = 

In [ ]:
#@title  {display-mode: "form"}
import sympy
W=sympy.symbols("W") #unit "Watt"
h=sympy.symbols("h") #unit "hour"
yr=sympy.symbols("yr") #unit "Year"
m=sympy.symbols("m") #unit "meter"
EpAY= 200e3*W*h/(m**2)/yr #Energy per area per year
EpY=209e12*W*h/yr #desired Energy per year
print(" = ",EpY,"/",EpAY, " =", EpY/EpAY)
#print("Flaechenbedarf ist Energie pro Jahr geteilt durch Energie pro Fläche pro Jahr:") )

 =  209000000000000.0*W*h/yr / 200000.0*W*h/(m**2*yr)  = 1045000000.0*m**2


* Prüfung: Die resultierende Einheit (m²) ist korrekt ("m**2" bedeutet m² in Python)
* 10⁶ m² sind 1 Quadratkilometer (km²)
* Dies sind also 1045 km². 
* Diese Fläche liesse sich leicht z.B. in den dann **stillgelegten deutschen Braunkohletagebauten** unterbringen, denn: 
* "Der Flächenverbrauch der deutschen Braunkohletagebauten betrug mit Stand 2006 mehr als **2300 km²**", sh.:

In [ ]:
%%html
<iframe src="https://de.wikipedia.org/wiki/Braunkohlebergbau" width="900" height="600"></iframe>

* Die nötigen Solarzellen liessen sich aber auch auf **Dächern** installieren
* Ein grosser Vorteil grade von Sonnenenergie ist ihre **Dezentralität**. Häuser lassen sich damit Energieautark betreiben.
* Fuer Solarzellen geeignete **Dachflaechen** in Deutschland summieren sich auf ca. 800 Mio. qm, nach
“Erneuerbare Energieträger in Deutschland: Potentiale und Kosten”,
edited by Martin Kaltschmitt, Andreas Wiese, 1993,
ISBN 978-3-642-93541-1, 
* 800 Mio. qm sind **800 qkm**, die Flaeche ist also fast benötigten 1045 qkm.


## Flächenbedarf Windenergie
<!--
**Füllung der andere Hälfte der Lücke von 56.5 TWh/Jahr durch Windenergie:**
* Flaechenbedarf Windenergie. 
-->

* [Umweltbundesamt, Potential der Windenergie](https://www.umweltbundesamt.de/sites/default/files/medien/378/publikationen/potenzial_der_windenergie.pdf)
* Kap. 4.1, S.16: " wurde hier vereinfachend von einem radialen Anlagenabstand
in Höhe des 4-fachen Rotordurchmessers ausgegangen. Maßgeblich bei der Platzierung
ist hierbei der Rotordurchmesser der Schwachwindanlage. Der radiale Anlagenabstand
beträgt damit **456 m**."
* Bei einer quadratischen Anordnung entspricht dies einer Fläche von **456² qm =207936 qm**
* Seite 15: Leistung je Windenergie Anlage: **3.2 MW = 3.2e6 W**
* In der Zusammenfassung, Seite 2: " ... im
bundesweiten Mittel **2440 Volllaststunden pro Jahr** ..."
<!-- * **Ergebnis:** Energie pro Jahr pro qm = **37.55 kWh/qm pro Jahr** -->



## Windkraft Energie pro qm pro Jahr
Energie pro qm pro Jahr = Leistung mal Vollaststunden pro Jahr geteilt durch Flaeche =  

In [ ]:
#@title  {display-mode: "form"}
import sympy
m=sympy.symbols("m") #unit "meter"
W=sympy.symbols("W") #unit "Watt"
h=sympy.symbols("h") #unit "hour"
yr=sympy.symbols("yr") #unit "Year"
P=3.2e6*W #Power of win turbine
fr=2440*h/yr # Hours per year of this power (fraction)
A=456**2*m**2 #Area covered by wind turbine
EpAYwind=P*fr/A
print(P,"*", fr, "/", A," = ", EpAYwind)
#print("Die Energie in Wh geteilt durch die Stunden im Jahr ergibt die Leistung, in Watt pro qm ")

3200000.0*W * 2440*h/yr / 207936*m**2  =  37550.0153893506*W*h/(m**2*yr)


Also 37550 Wh pro Jahr pro qm.
## Nötige Fläche für 209 TWh Windkraft pro Jahr:
Nötige Fläche = benötigte Energie pro Jahr geteilt durch Energie pro Jahr pro qm = 

In [ ]:
#@title {display-mode: "form"} 
print(" = ", EpY, "/", EpAYwind, " = ",  EpY/ EpAYwind)

 =  209000000000000.0*W*h/yr / 37550.0153893506*W*h/(m**2*yr)  =  5565909836.06557*m**2


* Dies ist ca. 5566 quadratkilometer.
* Die Gesamtfläche Deutschlands ist 357386 qkm, die zusätzlich nötige Fläche für Windkraft ist also ca. 1.6% davon. 
* Ca. die Hälfte der Fläche Deutschlands ist Ackerfläche, d.h. es wäre zusätzlich nur ca. 3% der Ackerfläche.

## Schlussfolgerungen
* Die benötigte erneuerbare Energie für den prognostizierten Bedarf in 2030 lässt sich **praktikabel im Land erzeugen**.
* Für die Fläche der Solarzellen würden die stillgelegten **Braunkohletagebauten** mehr als ausreichen.
* Die **geeigneten Dachflächen** würden fast vollständig ausreichen
* Fuer die Windkraft reichen zusätzlich **3% der Ackerfläche**, die aber trotzdem noch für Ackerbau zur Verfügung steht.


* Zum Ausgleich der Schwankungen von Wind und Sonne werden Speicher gebraucht
* Mehr dazu in einer folgenden Episode

<!--
* Durschnittsleistung neuer Windanlagen: 3.34 MW
* Nutzungsgrad: 16-57%
* "Moderne Schwachwindanlagen haben mittlerweile Rotordurchmesser bis über 140 Meter und Nabenhöhen bis über 160 Meter."
-->
<!---* "Dauerhaft muss für eine aktuelle Windkraftanlage der Drei-Megawatt-Klasse eine befestigte aber unversiegelte Fläche von etwa 2.500 m² für die Wartung frei und zugänglich bleiben. " --->

<!--
Mittlerer Abstand: 5*140 m 
Mittlerer Nutzungsgrad: 0.3
Also Leistung pro Fläche...
-->